### ISSUE TITLE
Feign降级的支持粒度，能否支持服务粒度(Can feign support service granularity)

### ISSUE THREAD

**Issue by y934317258** [alibaba/Sentinel#2161]
## Issue Description
Feign限流降级好像只支持url粒度，能否支持服务粒度，就像对RestTemplate支持两种粒度一样(Feign's current limiting degradation seems to only support URL granularity. Can it support service granularity, just like resttemplate supports two kinds of granularity
![image](https://user-images.githubusercontent.com/68832358/115989330-66c73f00-a5f0-11eb-8848-7321a150b83e.png)

Type: *bug report* or *feature request*

### How to reproduce it (as minimally and precisely as possible)

1. 
2. 
3.

**Comment by sczyh30** [comment]
Good idea. You may submit an issue in https://github.com/alibaba/spring-cloud-alibaba
Contributions are welcomed!

**Comment by y934317258** [comment]
Thank you. I have submitted it
